src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const marriedMan = person.isMarried && person.sex === 'm';
+  const marriedWoman = person.isMarried && person.sex === 'f';
+  
+  return (
+  <div className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
```

Comment:
Not fixed 
<img width="923" alt="image" src="https://github.com/user-attachments/assets/2f81e562-80ff-4caf-b3d3-d3fe49355dc9">